schema/swagger.yml

```diff
@@ -1631,7 +1631,7 @@ definitions:
         type: string
         format: signature
         example: 2821d93a742c4edf5fd960efad41a4def7bf0fd0f7c09869aed524f6f52bf9c97a617095e2c712bd28b4279078a29509b339ac55187854006591aa759784c205
-      multisignatures:
+      signatures:
```

Comment:
It may caused confused to differentiate `signature`, so why now I remember it was named as `multisignatures` explicitly. Point is items in `multisignatures` array are completely different than `signature`concept, so why it is named like this.

Reply:
There is no complexity, just the matter of mixing in concept of transaction signature and signatures of multisig account.

Rest upto @karmacoma to decide what's best approach he think is.

Reply:
Agree it can be confusing, but to be consistent with the transaction schema and how that schema is currently interpreted by the transaction pool, we will accept this PR. There will be another opportunity to revisit this after `1.0.0`.